modules/services/window-managers/hyprland.nix

```diff
@@ -175,22 +175,34 @@ in {
       shouldGenerate = cfg.systemdIntegration || cfg.extraConfig != ""
         || combinedSettings != { };
 
-      toHyprconf = attrs:
+      toHyprconf = with lib;
+        attrs: indentLevel:
         let
+          indent = concatStringsSep "" (replicate indentLevel "  ");
+
           mkSection = n: attrs: ''
-            ${n} {
-              ${toHyprconf attrs}}
+            ${indent}${n} {
+            ${toHyprconf attrs (indentLevel + 1)}${indent}}
           '';
-          mkFields = lib.generators.toKeyValue { listsAsDuplicateKeys = true; };
-          sections = lib.filterAttrs (n: v: lib.isAttrs v) attrs;
-          fields = lib.filterAttrs (n: v: !(lib.isAttrs v)) attrs;
-        in lib.concatStringsSep "\n" (lib.mapAttrsToList mkSection sections)
-        + mkFields fields;
+          sections = filterAttrs (n: v: isAttrs v) attrs;
+
+          mkFields = generators.toKeyValue { listsAsDuplicateKeys = true; };
+          allFields = filterAttrs (n: v: !(isAttrs v)) attrs;
+          importantFields =
+            filterAttrs (n: _: (hasPrefix "$" n) || (hasPrefix "bezier" n))
+            allFields;
```

Comment:
Could hyprland possibly be made order-agnostic in the future?

Reply:
I will ask.

Reply:
Yes, it could be made order-agnostic, but not with the current config parser, which processes the file line-by-line.